packages/@aws-cdk/aws-batch-alpha/lib/ecs-container-definition.ts

```diff
@@ -912,11 +926,18 @@ export interface EcsFargateContainerDefinitionProps extends EcsContainerDefiniti
 export class EcsFargateContainerDefinition extends EcsContainerDefinitionBase implements IEcsFargateContainerDefinition {
   public readonly fargatePlatformVersion?: ecs.FargatePlatformVersion;
   public readonly assignPublicIp?: boolean;
+  public readonly ephemeralStorageSize?: Size;
 
   constructor(scope: Construct, id: string, props: EcsFargateContainerDefinitionProps) {
     super(scope, id, props);
     this.assignPublicIp = props.assignPublicIp;
     this.fargatePlatformVersion = props.fargatePlatformVersion;
+    this.ephemeralStorageSize = props.ephemeralStorageSize;
+
+    // validates ephemeralStorageSize is within limits
+    if (props.ephemeralStorageSize && (props.ephemeralStorageSize.toGibibytes() > 200 || props.ephemeralStorageSize.toGibibytes() < 21)) {
+      throw new Error('Ephemeral Storage Size must be within 21 to 200 GiB.');
```

Comment:
something like this instead:

```
ECS Fargate container '${id}' specifies 'ephemeralStorageSize' at ${props.ephemeralStorageSize} > 200 GB
```

```
ECS Fargate container '${id}' specifies 'ephemeralStorageSize' at ${props.ephemeralStorageSize} < 21 GB
```

The rest of the errors in Batch are like this, so let's keep the style consistent.